script_library/deployment-actions-openstack.sh

```diff
@@ -36,66 +37,6 @@ function enroll_caasp_workers() {
     echo "Enrolling caasp worker nodes into the cluster"
     run_ansible ${socok8s_absolute_dir}/playbooks/generic-enroll_caasp_workers.yml
 }
-function patch_upstream(){
-    echo "Running dev-patcher"
-    echo "Nothing will happen if developer mode is not set"
-    run_ansible ${socok8s_absolute_dir}/playbooks/openstack-patch_upstream.yml
```

Comment:
This playbook (like some of the others following) just calls the generic playbook. We should remove the file if they are no longer needed.

Reply:
OK